packages/dashboard-backend/src/devworkspace-client/services/api/serverConfigApi.ts

```diff
@@ -21,8 +21,8 @@ const GROUP = 'org.eclipse.che';
 const VERSION = 'v1';
 const PLURAL = 'checlusters';
 
-const NAME = process.env.CHE_CRD_OBJECT_NAME;
-const NAMESPACE = process.env.CHE_NAMESPACE;
+const NAME = process.env.CHECLUSTER_CR_NAME;
+const NAMESPACE = process.env.CHECLUSTER_CR_NAMESPACE;
```

Comment:
Renamed to match env variable names from the che-operator:  https://github.com/dkwon17/che-operator/blob/051d7bd9d30d1b48a9752b095426c39f2b19aee1/pkg/deploy/dashboard/deployment_dashboard.go#L59-L64